angr/procedures/msvcr/_initterm.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+import angr
+from angr.sim_type import SimTypeInt
+
+l = logging.getLogger("angr.procedures.msvcr._initterm")
+
+######################################
+# __initterm
+######################################
+
+class _initterm(angr.SimProcedure):
+
+    IS_FUNCTION = True
+    local_vars = ['callbacks']
+
+    #pylint:disable=arguments-differ
+    def run(self, fp_a, fp_z):
+        self.argument_types = {0: self.ty_ptr(SimTypeInt()),
+                               1: self.ty_ptr(SimTypeInt())
+        }
+        self.return_type = SimTypeInt()
+
+        if any(map(self.state.solver.symbolic, [fp_a, fp_z])):
+            l.warn("Symbolic argument to _initterm{_e} is not supported... returning")
+            return 0 # might as well try to keep going
+
+        self.callbacks = self.get_callbacks(fp_a, fp_z) # pylint:disable=attribute-defined-outside-init
```

Comment:
The preferred way to deal with this is to add a class variable `callbacks = None` instead of hiding the pylint message